src/asciidoctor-kroki.js

```diff
@@ -73,11 +73,12 @@ const processKroki = (processor, parent, attrs, diagramType, diagramText, contex
     if (diagramType === 'vegalite') {
       diagramText = require('./preprocess.js').preprocessVegaLite(diagramText, context, diagramDir)
     } else if (diagramType === 'plantuml' || diagramType === 'c4plantuml') {
-      const plantUmlInclude = doc.getAttribute('kroki-plantuml-include')
-      if (plantUmlInclude) {
-        diagramText = `!include ${plantUmlInclude}\n${diagramText}`
+      const plantUmlIncludeFile = doc.getAttribute('kroki-plantuml-include')
+      if (plantUmlIncludeFile) {
+        diagramText = `!include ${plantUmlIncludeFile}\n${diagramText}`
       }
-      diagramText = require('./preprocess.js').preprocessPlantUML(diagramText, context, diagramDir)
+      const plantUmlIncludePaths = doc.getAttribute('kroki-plantuml-include-paths')
```

Comment:
should we use a default value if `kroki-plantuml-include-paths` is undefined. If `kroki-plantuml-include-paths` is not defined, `doc.getAttribute` will return `undefined`.

Reply:
I think it's OK, because it's checked in the function
`const includePaths = diagramIncludePaths ? diagramIncludePaths.split(path.delimiter) : []`
and looks good in debugger, because the array is empty then...

Reply:
Fair enough, `undefined` is indeed `falsy` :+1: